### PR TITLE
ENH: make MiniCPM v2.6 support video

### PR DESCRIPTION
**Pull request by Minamiyama**
https://github.com/user-attachments/assets/c42c9157-b4ab-4f1e-bdfe-7225bb34ab03

### CODE REVIEW

**Comment by qinxuye** [comment]
GPU CI failed: https://github.com/xorbitsai/inference/actions/runs/10382926517/job/28746934032?pr=2068

Looks like new package introduced? Could you remove the import into the function, refer to 

https://github.com/xorbitsai/inference/blob/894c9642c61bac04781ceb3379e3da7303e45418/xinference/model/image/stable_diffusion/core.py#L98-L105

**Comment by Minamiyama** [comment]
> GPU CI failed: https://github.com/xorbitsai/inference/actions/runs/10382926517/job/28746934032?pr=2068
> 
> Looks like new package introduced? Could you move the import into the function, refer to
> 
> https://github.com/xorbitsai/inference/blob/894c9642c61bac04781ceb3379e3da7303e45418/xinference/model/image/stable_diffusion/core.py#L98-L105

done

**Comment by qinxuye** [comment]
https://github.com/xorbitsai/inference/actions/runs/10399318614/job/28798033361?pr=2068

Looks like there is installation issue for Mac.

**Comment by Minamiyama** [comment]
> https://github.com/xorbitsai/inference/actions/runs/10399318614/job/28798033361?pr=2068
> 
> Looks like there is installation issue for Mac.

all done

File: xinference/core/chat_interface.py [hunk 1]
```diff
@@ -257,16 +257,48 @@ def add_text(history, bot, text, image):
                         },
                     ],
                 }
+            elif video:
+                # 读取视频文件并转换为Base64
```
Comment (qinxuye) [inline xinference/core/chat_interface.py:261]: Could you use English to comment here?

File: xinference/core/chat_interface.py [hunk 2]
```diff
@@ -257,16 +257,48 @@ def add_text(history, bot, text, image):
...
+                def video_to_base64(video_path):
+                    with open(video_path, "rb") as video_file:
+                        encoded_string = base64.b64encode(video_file.read()).decode(
+                            "utf-8"
+                        )
+                    return encoded_string
+
+                # 将视频路径转换为Base64并嵌入HTML中
```
Comment (qinxuye) [inline xinference/core/chat_interface.py:269]: Ditto.

**Review by qinxuye — APPROVED** [review]
LGTM

**Comment by qinxuye** [comment]
Could you add the library to 

https://github.com/xorbitsai/inference/blob/main/xinference/deploy/docker/requirements.txt

and 

https://github.com/xorbitsai/inference/blob/main/xinference/deploy/docker/requirements_cpu.txt

To ensure it's included in docker.

**Pull request merged**